allauth/account/tests.py

```diff
@@ -301,27 +330,6 @@ def test_ajax_login_success(self):
         data = json.loads(resp.content.decode('utf8'))
         self.assertEqual(data['location'], '/accounts/profile/')
 
-    @override_settings(
-        ACCOUNT_EMAIL_VERIFICATION=app_settings.EmailVerificationMethod
-        .MANDATORY,
-        ACCOUNT_AUTHENTICATION_METHOD=app_settings.AuthenticationMethod
```

Comment:
Please override the `ACCOUNT_AUTHENTICATION_METHOD` method as well, as you are logging in by username.